docs/geedocs/answer/3230746.html

```diff
@@ -84,67 +84,35 @@
   <h2><a id="InstallPortable" name="InstallPortable">Install GEE Portable</a></h2>
   <p>GEE Portable is supported on:</p>
   <ul>
-    <li>Windows XP and 7</li>
-    <li>Windows 2008 Server</li>
-    <li>Mac OS X 10.4 and later</li>
-    <li>Red Hat Enterprise Linux versions 5.6 to 6.2, including the most recent security patches</li>
-    <li>Ubuntu 10.04 LTS and 12.04 LTS</li>
+    <li>Windows 7 and 10</li>
+    <li>Red Hat Enterprise Linux versions 6.9 and 7, including the most recent security patches</li>
```

Comment:
Hm. Maybe not, yet. Are we planning on supporting 6.9, or should I remove it from here.